stdeb.cfg

```diff
@@ -1,8 +1,7 @@
 [DEFAULT]
-Depends: python-argparse, python-setuptools, python-catkin-pkg (> 0.2.9), python-yaml
-Depends3: python3-setuptools, python3-catkin-pkg (> 0.2.9), python3-yaml
-Conflicts: python3-catkin-tools
+Depends3: python3-setuptools, python3-catkin-pkg-modules, python3-yaml, python3-osrf-pycommon
 Conflicts3: python-catkin-tools
-Suite: xenial yakkety zesty artful bionic cosmic disco eoan focal stretch buster
+Suite3: xenial yakkety zesty artful bionic cosmic disco eoan focal stretch buster
```

Comment:
I don't think this line change is necessary. The separate Suite3 config field is only needed when the release suites differ between Python 2 and Python 3.

I'm pretty certain that stdeb defaults to `unstable` if no suites are configured although I haven't confirmed that. Leaving this line as it was while adding the `No-Python2` field below should be sufficient to halt all further python2 releases of this package.

Reply:
Yes, you seem to be right. Thank you, I changed it back.